tests/rewards/test_reward_fn.py

```diff
@@ -0,0 +1,46 @@
+"""Tests `imitation.rewards.reward_function` and `imitation.rewards.serialize`."""
+
+import numpy as np
+import pytest
+
+from imitation.rewards import reward_function, serialize
+
+
+def _funky_reward_fn(obs, act, next_obs, done):
+    """Returns consecutive reward from 1 to batch size `len(obs)`."""
+    # give each environment number from 1 to num_envs
+    return (np.arange(len(obs))).astype("float32")
+
+
+obs = np.random.randint(0, 10, (64, 100))
```

Comment:
Module-wide constants should be in `ALL_CAPS`: https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations